syntax = "proto3";

service DemoService {
    rpc sayHi(DemoRequest) returns (DemoResponse);
}

message DemoRequest {
}

message DemoResponse {
}
